### ISSUE TITLE
Building free flavour fails

### ISSUE THREAD

**Issue by est31** [bottiger/SoundWaves#82]
Hi! The F-Droid build for the latest version (0.148.8) fails. The build log is: https://f-droid.org/wiki/page/org.bottiger.podcast/lastbuild_330

```
/home/vagrant/build/org.bottiger.podcast/app/src/free/java/org/bottiger/podcast/flavors/MediaCast/VendorMediaRouteCast.java:23: error: VendorMediaRouteCast is not abstract and does not override abstract method seekTo(long,boolean) in GenericMediaPlayerInterface
public class VendorMediaRouteCast extends GoogleCastPlayer implements IMediaCast {
       ^
```

It would be really nice if you could fix the issue. Thank you!

**Comment by bottiger** [comment]
Sure. I'll fix it at once. Expect a new version in an hour or two.

That said - I have another question for you, which you may or may not be able to help with. In my Goole play version I include some proprietary libraries from google.

`apply plugin: 'com.google.gms.google-services'`

Obviously this doesn't work on F-droid. However, I tried to only include it you do not compile the "free flavor" like this:

```
task nonfreeReleaseTask << {
    apply plugin: 'com.google.gms.google-services'
}

tasks.whenTaskAdded { task ->
    if (task.name != 'free') {
        task.dependsOn nonfreeReleaseTask
    }
}
```

But it didn't work. Is there any other way I can avoid editing this line every time I compile my project for Google/F-Droid. Because, I mess it up way to often.

**Comment by est31** [comment]
Sorry, I don't know gradle this close. Maybe @mvdan knows?

**Comment by bottiger** [comment]
@est31 No problem. In any case - I have pushed a fix in 0.148.10

**Comment by est31** [comment]
Thanks, it works now!
